prow/plugins/approve/approve_test.go

```diff
@@ -1292,6 +1324,7 @@ func TestHandleGenericComment(t *testing.T) {
 
 	for _, test := range tests {
 		test.commentEvent.Repo = repo
+		githubConfig := config.GitHubOptions{}
```

Comment:
So in ever case where we're passing an empty configuration let's be explicitly clear why we aren't testing the behavior

Reply:
(and that's the outlier, we should test in every case we can)

Reply:
It's better to test this functionality... updated and thank you :smile: